src/app/components/Auth/Auth.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { Tab, Tabs } from 'react-toolbox/lib/tabs';
+import Input from 'react-toolbox/lib/input';
+import Button from 'react-toolbox/lib/button';
+import styles from './Auth.scss';
+
+class Auth extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            index: 0,
+            email: '',
+            password: ''
```

Comment:
The component state won't persist once Auth unmounts. So it's not really different from a regular input field. Now, I didn't look to hard into this issue, so I'm probably wrong.